### PR TITLE
Add XING

### PR DESCRIPTION
**Pull request by AndreasDriesen**
Added Xing.com

### CODE REVIEW

**Comment by mxxcon** [comment]
Logo filename must not be "placeholder.png" and the logo itself must be added to your pull request, rather than adding it to a comment.

**Comment by AndreasDriesen** [comment]
I am not allowed to upload images in /img/social. Sorry for my newbie problems...

**Comment by mxxcon** [comment]
If you don't know how to add a file using git cli, this link explains how to upload files using Github's website https://help.github.com/articles/adding-a-file-to-a-repository/
Upload that logo into your fork's _patch-2_ branch.

**Comment by Carlgo11** [comment]
Hello @AndreasDriesen,
You're probably trying to upload the image to our repo instead of your fork.
Go to https://github.com/AndreasDriesen/twofactorauth/tree/patch-2/img/social and upload the image there instead. (Just drag and drop the image)

Let us know if you encounter any problem :)

File: _data/social.yml [hunk 1]
```diff
@@ -150,6 +150,12 @@ websites:
       exceptions:
           text: "SMS-capable phone required."
       doc: https://en.support.wordpress.com/security/two-step-authentication/
+      
+    - name: Xing.com
+      url: https://xing.com
+      img: placeholder.png
```
Comment (Carlgo11) [inline _data/social.yml]: Please change the image name.

File: _data/social.yml [hunk 2]
```diff
@@ -150,6 +150,12 @@ websites:
       exceptions:
           text: "SMS-capable phone required."
       doc: https://en.support.wordpress.com/security/two-step-authentication/
+      
+    - name: Xing.com
```
Comment (Carlgo11) [inline _data/social.yml]: The seem to use the name XING themselves. Maybe change the name to their correct name?

File: _data/social.yml [hunk 3]
```diff
@@ -150,6 +150,12 @@ websites:
...
+      url: https://xing.com
```
Comment (Carlgo11) [inline _data/social.yml]: Change the URL to `https://www.xing.com/`

**Pull request closed without merging**